berza/src/main/java/rs/edu/raf/banka/berza/bootstrap/BootstrapData.java

```diff
@@ -104,5 +111,29 @@ public void run(String... args) throws Exception {
             berze.add(berza);
         }
         berzaRepository.saveAll(berze);
+
+        //Dodavanje informacija o inflacijama
+        List<IstorijaInflacije> inflacije = new ArrayList<>();
+
+        List<InflacijaCSV> inflacijeCSV = new CsvToBeanBuilder(new FileReader(inflacijeCSVPath))
+                .withType(InflacijaCSV.class)
+                .withSkipLines(1)
+                .build()
+                .parse();
+
+        for(InflacijaCSV ic : inflacijeCSV) {
+            IstorijaInflacije istorijaInflacije = new IstorijaInflacije();
+            Optional<Valuta> valutaInflacije = valutaRepository.getValutaByNazivValute(ic.getCurrency());
+            if(valutaInflacije.isEmpty()) {
+                System.err.println("Preskacem inflaciju zato sto valuta " + ic.getCurrency() + " ne postoji!");
+                continue;
+            }
+            istorijaInflacije.setValuta(valutaInflacije.get());
+            istorijaInflacije.setYear(ic.getYear());
+            istorijaInflacije.setInflationRate(Double.parseDouble(ic.getInflationRate().substring(0,3)));
```

Comment:
Jel možemo samo da izbacimo procenat iz CSV-a?

Reply:
Moze da se izbaci iz CSV-a, svakako ga parsiramo ovde pri upisu u bazu, nego je procenat tu da ispostuje specifikaciju

Reply:
Nema potrebe striktno u reč da postuješ specifikaciju, slobodno izbaci to, biće mnogo bolje i sigurnije parsiranje.